pkg/controller/sparkapplication/controller.go

```diff
@@ -362,6 +375,12 @@ func (c *Controller) getAndUpdateAppState(app *v1beta1.SparkApplication) error {
 	return nil
 }
 
+func (c *Controller) getAndUpdateAppState(app *v1beta1.SparkApplication) error {
+	_ = c.getAndUpdateDriverState(app)
+	_ = c.getAndUpdateExecutorState(app)
```

Comment:
I could have simply written here `c.getAndUpdateExecutorState(app)` without the underscore and assignment. But my editor GoLand complained about "error not handled".

Reply:
You might want to do:
```
if err := c.getAndUpdateDriverState(app); err != nil {
    return err
}
if err := c.getAndUpdateExecutorState(app); err != nil {
    return err
}
```

Reply:
Thanks. Done.